### DISCUSSION TITLE
Option to disable round brackets (parenthesis) auto close

### DISCUSSION THREAD

**Discussion by adamitj** [dbeaver/dbeaver#383] [Q&A] [answered]
Using dBeaver v3.6.5 64-bit (Windows 10 64-bit Java 1.8.0_77)

I'm not native english speaker. Wikipedia[1] says that parenthesis are also a type of brackets (round brackets). If so, this must be a bug. If don't, this is a new feature request.

**if bug:**
Preferences --> General --> Editors --> SQL Editor --> GroupBox "Auto Close" --> CheckBox "Brackets:"
   Does not work for round brackets "(".

**if feature request:**
Add an option to disable auto close of parenthesis at:
Preferences --> General --> Editors --> SQL Editor --> GroupBox "Auto Close"

Thank you very much!

[1] [https://en.wikipedia.org/wiki/Bracket](url)

**Comment by r-richmond** [comment, marked as answer]
Have you tried unchecking brackets and then opening a new sql editor window? (I've noticed that sometimes you have to open a new sql editor window for preference changes)

**Comment by serge-rider** [comment]
Yep, exactly. You have to reopen SQL editor after config change.
In fact there are a lot of editor options, some of them takes effect immediately, some not. Reopening of editor helps in most cases.
